### PR TITLE
#43 Updated Search contacts

### PR DESCRIPTION
**Pull request by KennanHoa**
#43 

- check if users already in database.
* Contact requires the country code (i.e +1)

### CODE REVIEW

**Comment by refatK** [comment]
Please link the issue in you commit next time btw

File: .idea/gradle.xml
Comment (refatK) [inline .idea/gradle.xml:18]: Could you explain what this does?

File: app/src/main/java/com/example/bigowlapp/SearchContactsToSupervise.java
Comment (refatK) [inline app/src/main/java/com/example/bigowlapp/SearchContactsToSupervise.java:107]: You should probably remove these logs.

**Review by refatK — COMMENTED** [review]
I tried the application and it seems to works, although i'm not sure it was the expected result.

When I clicked on a contact without a number match the app would pop up a toast sayin the user already exists.

If the number exactly matched the number in the db, then it went to the sms invitation page.

* Is the number supposed to be auto filled when it swaps to sms invitation page, or must the user manually enter the number themself?

* In contacts the number might be something like `12345671234` and in db it could be `+12345671234`. I was assuming it would say it exists, but it said the umber didn't exist. Was that expected?

File: app/src/main/java/com/example/bigowlapp/SearchContactsToSupervise.java
Comment (refatK) [inline app/src/main/java/com/example/bigowlapp/SearchContactsToSupervise.java:105]: please reformat this file to use same style as everyone else.

You can do this in Android studio using the command `CTRL+ALT+SHIFT+L`, and selecting reformat code

**Review by KhangZhengNg — CHANGES_REQUESTED** [review]
Many files are in com.example.bigowlapp which is totally wrong. Please update the branch from staging and put the files in proper directory after com.example.bigowlapp. Check and remove code instead of commenting them out if they are useless. Also consider replacing listener with new method with lambda (suggestions from yellow light bulb)

**Review by refatK — APPROVED** [review]
Thanks for doing the changes I asked for. Approved!

File: app/src/main/java/com/example/bigowlapp/activity/SearchContactsToSupervise.java
Comment (KhangZhengNg) [inline app/src/main/java/com/example/bigowlapp/activity/SearchContactsToSupervise.java:98]: Nice and clean way to do query in FirebaseFirestore!

**Review by KhangZhengNg — APPROVED** [review]
Thanks for sticking with the project's folder structure. Good job!

**Review by darkheros12 — CHANGES_REQUESTED** [review]
Please, before merging, pull from staging to avoid any inconvenient. I will approve it once it is one.

**Pull request merged**
